### PR TITLE
Component/buttons

### PR DESCRIPTION
**Pull request by Scttpr**

### CODE REVIEW

File: src/components/atoms/Button/Button.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    setup(props) {
+        const classNames = computed(() => {
+            let list = 'font-body';
+
+            if (props.circular) {
+                list += ' rounded-full';
+            } else {
+                list += ' rounded';
+            }
```
Comment (Noloxe-Fred) [inline src/components/atoms/Button/Button.vue:30]:
```suggestion
            let list = 'font-body rounded';

            if (props.circular) {
                list += '-full';
            }
```

File: src/components/molecules/ProfileButton/ProfileButton.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<template>
+    <base-button class="py-4 px-6 flex items-center text-left">
+        <i
+            class="mr-4 rounded-full p-4 w-14 h-14 flex justify-center items-center"
+            :class="avatarDynamicClasses"
+            :style="avatarStyle"
+            >{{ avatarText }}</i
+        >
```
Comment (Noloxe-Fred) [inline src/components/molecules/ProfileButton/ProfileButton.vue]:
```suggestion
        >
            {{ avatarText }}
        </i>
```

File: src/components/molecules/ProfileButton/ProfileButton.vue [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        },
+    },
+    setup(props) {
+        const recipesNumberText = computed(() => {
+            let prefix = props.recipesNumber;
+            let suffix = 'recette publiée';
+
+            if (props.recipesNumber > 1) {
+                suffix = 'recettes publiées';
+            }
```
Comment (Noloxe-Fred) [inline src/components/molecules/ProfileButton/ProfileButton.vue]:
```suggestion
            const suffix = props.recipesNumber > 1 ? 'recette publiées' : 'recette publiée';
```

**Review by Djyp — APPROVED** [review]
Of course I approve :sweat_smile: 
I think @Noloxe-Fred's suggestions are nice but I'm not an expert in the required coding style here

File: src/components/molecules/ProfileButton/ProfileButton.vue (hunk 2 above)
Comment (Scttpr) [reply src/components/molecules/ProfileButton/ProfileButton.vue]: Thanks, I update

File: src/components/atoms/Button/Button.vue (hunk 1 above)
Comment (Scttpr) [reply src/components/atoms/Button/Button.vue:30]: I understand the proposal but I think it is better to keep the two separate classes to keep it clear. It is two different things with two different effects. Thanks for the proposal anyway !

File: src/components/molecules/ProfileButton/ProfileButton.vue (hunk 3 above)
Comment (Scttpr) [reply src/components/molecules/ProfileButton/ProfileButton.vue]: It is better to go for the ternary, thanks !

**Comment by Scttpr** [comment]
I updated according to PR comments and merged, thanks for your reviews !

**Pull request merged**